frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/form/form-field-review-row.tsx

```diff
@@ -18,16 +19,17 @@ export const FormFieldReviewRow: React.FC<FormFieldReviewRowProps> = ({
   field,
   value = undefined,
 }) => {
+  const { t } = useTranslation();
   const asId = prefixedID.bind(null, 'wizard-review');
   const fieldKey = iGet(field, 'key');
   const fieldTitle = getFieldTitle(fieldKey);
   const reviewValue = value || getReviewValue(field, fieldType) || (
-    <span className="text-secondary">{`No ${fieldTitle.toLowerCase()}`}</span>
+    <span className="text-secondary">{t('kubevirt-plugin~Not defined')}</span>
```

Comment:
text changed ... is it on purpose ?

Reply:
I wanted to avoid msg construction so I changed it to simply `Not defined` which works just fine IMO

Reply:
+1